orc8r/cloud/go/services/bootstrapper/const.go

```diff
@@ -13,4 +13,15 @@ limitations under the License.
 
 package bootstrapper
 
-const ServiceName = "BOOTSTRAPPER"
+const (
+	ServiceName        = "BOOTSTRAPPER"
+	BlobstoreTableName = "bootstrapper"
+	TokenPrepend       = "reg_"
+)
+
+type DBBlobType string
```

Comment:
The fact that this backs to a DB is an impl detail

```suggestion
type BlobType string
```